pkg/config/config.go

```diff
@@ -29,6 +29,7 @@ type Serialized struct {
 		Interval string `json:"interval"`
 		Disabled bool   `json:"disabled"`
 	}
+	ConditionalGathererRulesEndpoint string `json:"conditional_gatherer_rules_endpoint"`
```

Comment:
`json:"conditional_gatherer_rules_endpoint"`doesn't correspond to the [config in the yaml files](https://github.com/openshift/insights-operator/pull/483/files#diff-073e48cb952955fd3f328a1110efe362d38fad66b985b21c5b1a1865425312c1R19) and thus it won't work.

Reply:
fixed